client/src/App.tsx

```diff
@@ -0,0 +1,102 @@
+import { useEffect } from "react";
+import reactLogo from "./assets/react.svg";
+import viteLogo from "/vite.svg";
+import "./App.css";
+import TodoForm from "./TodoForm";
+import TodoItem from "./TodoItem";
+import Pagination from "./Pagination";
+import { useDispatch, useSelector } from "react-redux";
+import type { AppDispatch, RootState } from "./store/store";
+import {
+  deleteTodoThunk,
+  fetchTodosThunk,
+  setLimit,
+  setPage,
+  toggleTodoThunk,
+  updateTodoTextThunk,
+} from "./store/todoSlice";
```

Comment:
```suggestion
import {
  deleteTodoThunk,
  ....
} from "@/store/todoSlice";
```
Надо импорты сделать через allias @/ = ./src 
1 настройка, что бы пути красиво выглядили

Reply:
я хз как это сделать, попробовал покопаться в vite.config.ts, ничего не получилось

Reply:
https://stackoverflow.com/questions/77249074/how-do-i-use-typescript-path-aliases-in-vite#:~:text=There%27s%20a%20plugin%20for%20that%3A%20vite%2Dtsconfig%2Dpaths.%20You%20run